tsconfig.json

```diff
@@ -11,7 +11,6 @@
     "module": "esnext",
     "target": "es2018",
     "lib": ["es6", "esnext", "es2015", "dom"],
-    "suppressImplicitAnyIndexErrors": true,
```

Comment:
I went ahead with removing this option as it's been deprecated. From the TS issue:

```
suppressImplicitAnyIndexErrors
This flag disables the error produced when indexing an object which doesn't support indexing.
This was always intended to be a temporary flag, so is going away.

Fix: Add an index signature to the relevant type (or use a type assertion at the indexing location).
```